services/src/main/java/org/broadleafcommerce/payment/service/gateway/PayPalCheckoutRestConfiguration.java

```diff
@@ -0,0 +1,173 @@
+     * payment or billing agreement.
+     *
+     * Character length and limitations: 2048 single-byte characters
+     */
+    @Getter(AccessLevel.NONE)
+    @Setter(AccessLevel.NONE)
+    private String returnUrl;
+
+    /**
+     * URL to which the buyer is returned if the buyer does not approve the use of PayPal to pay
+     * you. For digital goods, you must add JavaScript to this page to close the in-context
+     * experience. Note: PayPal recommends that the value be the original page on which the buyer
+     * chose to pay with PayPal or establish a billing agreement.
+     *
+     * Character length and limitations: 2048 single-byte characters
+     */
+    @Getter(AccessLevel.NONE)
+    @Setter(AccessLevel.NONE)
+    private String cancelUrl;
+
+    /**
+     * Identifies whether production ("live") or non-production/test ("sandbox") transactions should
+     * be made.
+     *
+     * @return the execution mode (sandbox vs live) of transactions through the PayPal API
+     */
+    private String mode = "sandbox";
+
+    /**
+     * The WebProfile to be used when creating payments. For more information on WebProfiles go to
+     * {@link https://developer.paypal.com/docs/integration/direct/payment-experience/}.
+     * {@link PayPalWebProfileService#getWebProfileId(PaymentRequest)} should be used instead if you
+     * want to find the web profile id to create a payment since it has the ability to create new
+     * WebProfiles based on injected beans along with using this method
+     *
+     * @return The WebProfile to be used when creating payments
+     */
+    private String webProfileId;
+
+    /**
+     * The client id provided by PayPal that is used in API calls to identify the charging
+     * company/entity.
+     *
+     * @return The client id provided by PayPal that is used in API calls to identify the charging
+     *         company/entity.
+     */
+    private String clientId;
+
+    /**
+     * The client secret provided by PayPal that is used in API calls to verify the applications
+     * usage of the client id.
+     *
+     * @return The client secret provided by PayPal that is used in API calls to verify the
+     *         applications usage of the client id.
+     */
+    private String clientSecret;
+
+    /**
+     * Simple description of the entity that is charging the customer. For example, this may read as
+     * "My Test Store".
+     *
+     * Note, this is the transaction description that will be listed on the customer's credit card
+     * statement.
+     *
+     * @return Simple description of the entity that is charging the customer
+     */
+    private String paymentDescription;
+
+    /**
+     * Type declaration for the label to be displayed in MiniCart for UX. It is one of the following
+     * values: - Total - EstimatedTotal
+     *
+     * @return String
+     */
+    private String totalType = MessageConstants.TOTAL;
+
+    /**
+     * <p>
```

Comment:
Do not use any Javadoc formatting it does not come through. You can test this by doing autocompletion of properties in your IDE, any Javadoc formatting does not show up correctly.